frappe/utils/data.py

```diff
@@ -260,6 +260,17 @@ def global_date_format(date, format="long"):
 	formatted_date = babel.dates.format_date(date, locale=(frappe.local.lang or "en").replace("-", "_"), format=format)
 	return formatted_date
 
+def is_weekday(date):
+	date = getdate(date)
+	return date.weekday() not in [5, 6]
+
+def next_weekday_date(date):
+	date = getdate(date)
+	date = add_days(date, 1)
+	while is_weekday(date) is False:
+		date = add_days(date, 1)
+	return date
```

Comment:
if it takes a string date, shouldn't it return a string date as well?

Reply:
Sure, that makes sense.